ui/__tests__/components/policy/req.test.js

```diff
@@ -1,19 +1,21 @@
-import { shallow } from 'enzyme';
+import { shallow, mount } from 'enzyme';
 import React from 'react';
 
 import Req from '../../../components/policy/req';
 
 describe('<Req />', () => {
+  const req = {
+    req_id: '123.45',
+    req_text: 'Some text here',
+    topics: [],
+    policy: [],
+  };
+
   it('includes meta data when highlighted', () => {
-    const req = {
-      req_id: '123.45',
-      req_text: 'Some text here',
-      topics: [],
-    };
-    const normalText = shallow(
+    const normalText = mount(
       <Req href="" onClick={jest.fn()} req={req} />,
     ).text();
-    const hlText = shallow(
+    const hlText = mount(
```

Comment:
Also small: `mount` renders a component and all of its children where `shallow` will only render the component; that makes `shallow` a bit better for pure unit tests (as we aren't depending on the work of contained components). Where possible, we should prefer `shallow` for that reason.

Reply:
Updated